packages/runtime/src/widgets/Dropdown.tsx

```diff
@@ -47,6 +50,7 @@ const Dropdown: React.FC<DropdownProps> = (props) => {
       }}
     >
       <Select
+        defaultValue={String(values?.value)}
```

Comment:
Not sure why this is needed?

Reply:
defaultValue is needed in case of dropdown for enrollment screen to show a program which was being selected at selectDOS screen, it was not working before so i had to add it explicitly.

Reply:
In that case, you would just set the value? Default value should only apply if no value is present.